applications/dashboard/views/email/src/email-basic.php

```diff
@@ -192,7 +192,7 @@
     </style>
     <!--[if (gte mso 9)|(IE)]>
     [[literal]]
-    <style type="text/css">
+    <style>
```

Comment:
Some email clients are not yet HTML5 compliant, so I don't feel comfortable removing this yet.